### PR TITLE
xsp: add page

### LINKED ISSUE #7899: page request: xsp
[Mono main tools](https://www.mono-project.com/docs/tools+libraries/tools/#main-tools)

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

closes #7899

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages/common/xsp.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# xsp
+
+> Mono ASP.NET Web Server.
+> More information: <https://www.mono-project.com/docs/web/aspnet/>.
+
+- Use a specific IP to listen on:
```
Comment (navarroaxel) [inline pages/common/xsp.md:6]: I prefer something like this
```suggestion
- Start and bind to a specific IP address (defaults to `0.0.0.0`):
```
by default is `127.0.0.1` or `0.0.0.0`?

File: pages/common/xsp.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`xsp --address {{0.0.0.0}}`
+
+- Use a specific port to listen on:
```
Comment (navarroaxel) [inline pages/common/xsp.md:10]:
```suggestion
- Start and listen a specific port:
```

File: pages/common/xsp.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/common/xsp.md:6]: According to [manual](https://manned.org/xsp.1#head4) by default it's `0.0.0.0`.

> By default it's 0.0.0.0 for xsp
              and 127.0.0.1 for mod-mono-server.

File: pages/common/xsp.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# xsp
+
+> Mono ASP.NET Web Server.
+> More information: <https://www.mono-project.com/docs/web/aspnet/>.
+
+- Use a specific IP to listen on:
+
+`xsp --address {{0.0.0.0}}`
+
+- Start and listen a specific port:
```
Comment (pixelcmtd) [inline pages/common/xsp.md:10]:
```suggestion
- Start and listen on a specific port:
```

**Pull request closed without merging**